skypeweb/purplecompat.h

```diff
@@ -28,6 +28,12 @@
 //TODO remove this when dx adds this to the PurpleMessageFlags enum
 #define PURPLE_MESSAGE_REMOTE_SEND  0x10000
 
+static inline void purple_conversation_write_system_message_ts(PurpleConversation *conv, const gchar *msg, PurpleMessageFlags flags, time_t ts) {
```

Comment:
In fact, it'd be better if the code that used `purple_conversation_write_system_message_ts` in skypeweb_contacts just used the three lines of code that this function uses instead